client/cached.js

```diff
@@ -2,6 +2,14 @@ const initCACCL = require('.');
 
 const cacclInstanceCache = {}; // jsonified config => CACCL instance
 
+/**
+ * Memoized version of the initialization function. If given the same config
+ *   options, this function will return the previously initialized caccl
+ *   instance
+ * @author Gabe Abrams
+ * @param {object} config - the same arguments as allowed in /client/index.js
+ * @return {object} same response as /client/index.js
+ */
 module.exports = (config = {}) => {
   const cacheKey = JSON.stringify(config);
```

Comment:
Not related to these changes but it seems possible for this to lead to cache misses if the json doesn't alwasy serialize in the same order. Searching npmjs shows a few "deterministic" stringify packages: https://www.npmjs.com/search?q=stable%20stringify. Just a thought.

Reply:
Great point! Changed!